references/dump.md

```diff
@@ -0,0 +1,46 @@
+# Dump
+
+The `dump` route allow the creation of database dumps. Dumps are compatible between MeiliSearch versions.
```

Comment:
Can you specify what a dump contains? (indexes, settings, NOT updates, etc...)
:)